src/metabase/task.clj

```diff
@@ -97,7 +97,7 @@
                        (ex-message (.getCause e)))
             (qs/delete-job scheduler job-key)))))))
 
-(defn- init-scheduler!
+(defn init-scheduler!
```

Comment:
Assume you've made sure that `(init-scheduler!)` is idempotent right? So we can call it once, then `(start-scheduler!)` will call it a second time and that's ok?

Reply:
Yes `init-scheduler!`'s body only runs when the scheduler atom is [non-nil](https://github.com/metabase/metabase/blob/master/src/metabase/task.clj#L105)